src/DetailsView/reports/automated-checks-report.scss

```diff
@@ -34,6 +34,10 @@ body {
         font-size: 17px;
         line-height: 24px;
     }
+
+    @media only screen and (max-width: 1000px) {
```

Comment:
where did we get the `1000px` value? is it on figma?

Reply:
It is when the border of the screen meet the content plus margin.